### ISSUE TITLE
Choppy scrolling

### ISSUE THREAD

**Issue by Mrnofish** [flackbash/AudioAnchor#179]
Scrolling the list of chapters for an already synchronized folder is choppy.

On my device (Snapdragon 855) the issue aggravates when the folder has a few dozen chapters, and becomes decidedly obvious with larger folders (50+ chapters).

By using split screen, I can notice a slight choppiness even with folders that contain less than 10 chapters.

**Comment by flackbash** [comment]
Yes, this is related to #26, although the cover images don't seem to be the only or main issue.
